test/tfvc/commands/findworkspace.test.ts

```diff
@@ -0,0 +1,81 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+"use strict";
+
+import { FindWorkspace } from "../../../src/tfvc/commands/findworkspace";
+import { TfvcError } from "../../../src/tfvc/tfvcerror";
+import { IExecutionResult, IWorkspace } from "../../../src/tfvc/interfaces";
+
+var chai = require("chai");
```

Comment:
Either before or after going in, we should make the same change as in my last PR (import chai).